Makefile

```diff
@@ -143,3 +143,8 @@ crd:
 # Check go format
 check-format: 
 	if [ ! -z "$(FORMAT_LIST)" ] ; then echo invalid format at: ${FORMAT_LIST} && exit 1; fi
+
+generate-helm: wego
+	kind create cluster --name wego-helm-generator
```

Comment:
It would be nice to save the current kubectl context and set it back at the end of this make target. Otherwise, the user has their kubectl pointing to a cluster that no longer exists.